Automattic/newspack-blocks
# Update block icons to use Gutenberg style instead of Material

### All Submissions:






### Changes proposed in this Pull Request:

* Update calypso-build package
* Remove material-ui packages
* Add wordpress icons package
* Use a combination of custom icons and wp icons for the various blocks (inc. toolbar)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test the various Newspack blocks, do you see the new icons

### Other information:
